### ISSUE TITLE
CNV and LOH analysis for TSO500

### ISSUE THREAD

**Issue by ningb** [hartwigmedical/hmftools#563]
To whom it may concern,

Thank you for developing such wonderful tools! We are trying to use the pipeline to process our TSO500 panel data, and it would be greatly appreciated if you could answer some of our concerns regarding CNV analysis:

- In the output from PURPLE, the `purple.cnv.gene.tsv` file contained all genes, rather than the ~500 genes in the panel. Is this expected? For our downstream analysis to identify CNV genes, should we simply filter down to the genes in the panel, and then filtered by `minCopyNumber`?
- We noticed there are some discrepancies between the output from the pipeline and the default output from TSO500 (DRAGEN software), particular for CNVs. The DRAGEN pipeline only allowed CNV calling in a small set of genes for sure, and LOH was not supported. So we wondered has the TSO500 application of `hmftools` been used before? Are there any publications we could reference? Or do you know maybe how we could validate the different calls?
- For the reference files specific to TSO500 such as `cobalt_normalisation.tso500.38.tsv` and `pon_artefacts.tso500.38.tsv.gz`, do you have a document for how they were generated?

We run the pipeline as it is for the targeted panel with all default parameters. 

Thank you!

Best

**Comment by charlesshale** [comment]
Yes it is expected, and yes you can filter them from the driver gene panel after Purple has run.

We have tested and validated a few 100 samples on TSO500, but not published these results.

The targeted panel training routine is described here:
https://github.com/hartwigmedical/hmftools/blob/master/pipeline/README_TARGETED.md
